browser/brave_content_browser_client_browsertest.cc

```diff
@@ -625,29 +626,6 @@ IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, MixedContentForOnion) {
 }
 #endif
 
-#if BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
-IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest,
-                       HangoutsDisabledByDefault) {
-  ASSERT_FALSE(browser()->profile()->GetPrefs()->GetBoolean(kHangoutsEnabled));
-  extensions::ExtensionRegistry* registry =
-      extensions::ExtensionRegistry::Get(browser()->profile());
-  ASSERT_FALSE(registry->enabled_extensions().Contains(hangouts_extension_id));
```

Comment:
how about converting this test into a new test with the "expected" list of extensions?

Reply:
Yes, I'll do that @goodov

Reply:
Added :+1: